assets/home/main.js

```diff
@@ -66,33 +66,59 @@ const Main = () => {
 	} else {
 		content = (
 			<>
-				<TasksSection data={ data.tasks } />
-
-				<Col as="section" className="sensei-home__section" cols={ 6 }>
-					<QuickLinks quickLinks={ data?.quick_links } />
-				</Col>
-
-				<Col as="section" className="sensei-home__section" cols={ 6 }>
-					<GetHelp categories={ data?.help } />
-				</Col>
-
-				<SenseiProAd />
-
-				<Col as="section" className="sensei-home__section" cols={ 6 }>
-					<SenseiGuides data={ data?.guides } />
-				</Col>
-
-				<Col as="section" className="sensei-home__section" cols={ 6 }>
-					<LatestNews data={ data?.news } />
-				</Col>
-
-				<Col
-					as="section"
-					className="sensei-home__section sensei-home__section__extensions"
-					cols={ 12 }
-				>
-					<Extensions />
-				</Col>
+				{ data?.tasks && <TasksSection data={ data.tasks } /> }
```

Comment:
Not directly related to this PR, but maybe we could update the `data` state to have a default value as an empty object `const [ data, setData ] = useState( {} );`, so we don't need the `?.` after all the `data`.

Not a critical thing though, in case you wanna skip this.

Reply:
Good change! Made in a2f5f4f207d63c50327529a77515579558627156

Reply:
Thank you for updating this! Looks good and works well!